.github/workflows/pr_ete_test.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         run: |
           python3 -m pip install cmake
           python3 -m pip install -r requirements/build.txt
+          sed -i 's/https:\/\/github.com\/NVIDIA\/cutlass.git/https:\/\/521github.com\/extdomains\/github.com\/NVIDIA\/cutlass.git/g' CMakeLists.txt
```

Comment:
will this work stable for the proxy?

Reply:
> will this url work steadily for the proxy?

I tried it works. And the speed is 10M+/s.